### ISSUE TITLE
Diminished mental health

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6728]
**Preferred term label:**
Mental health impairment

**Synonyms**
Reduced SF-36 mental health domain (MH) and mental health component summary
Mental impairment

**Definition (free text, please give PubMed ID)**
A reduction in the subjective feeling of mental well being. 

Comment: 
The WHO defines mental health as “a state of well-being in which the individual realizes his or her own abilities, can cope with the normal stresses of life, can work productively and fruitfully, and is able to make a contribution to his or her community”. Thus, mental health is not a state of absence of mental illness. Mental health has been defined on the basis of three components of mental health: emotional well-being, psychological well-being and social well-being. Alternatively, mental health can be defined as a dynamic state of internal equilibrium which enables individuals to use their abilities in harmony with universal values of society. Attempts to measure mental health in as objective a fashion as possible may involve questionnaires such as the 36-item Short Form (SF-36) health survey questionnaire. Impaired mental health is characterized by a reduction in the mental component score (MCS).

PMID: 22654934
PMID:26043341

**Parent term (use hpo.jax.org/app)**
Constitutional symptom HP:0025142

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
